### ISSUE TITLE
Edit email from user settings

### ISSUE THREAD

**Issue by ajefts** [appirio-tech/topcoder-app#911]
Add the ability to edit email from user settings page.

**Comment by fnisen** [comment]
Wanna be involved in this one, as well, please.

**Comment by ajefts** [comment]
Oh, you will be @fnisen ...don't worry :) I need to get ramped back up on where we left the state of profiles, settings, data sources of truth, etc... But at least this will spark that conversation again

**Comment by birdofpreyru** [comment]
@ajefts Is it in scope for the Bug Bash? I believe, this is mostly a work to be done on the backend side, and it is probably a big enough for a separate challenge, as just letting somebody to enter a new e-mail address is not secure. Some mechanism should be implemented, so that a notification is send to the old e-mail address to verify the change, etc.

**Comment by fnisen** [comment]
@birdofpreyru yeah, we'd need something like that. The experience could be similar to GitHub's: 
- You can not change an address per se
- You can add an email address that you then need to verify (from an email link)
- You can delete verified and unverified email addresses, but you cannot delete a verified email address if it's your last remaining one
